hlsq.go

```diff
@@ -71,9 +79,59 @@ func main() {
 		}
 		opts = append([]hlsqlib.SerializeOption{hlsqlib.AttrMatch(queryFunc)}, opts...)
 	}
-	for scanner.Scan() {
-		line := scanner.Tag()
-		hlsqlib.Serialize(os.Stdout, line, opts...)
+	if *watch && *url != "" {
+		opts = append(opts, hlsqlib.Chomp)
+		var updatePeriod time.Duration
+		seen := make(map[string]bool)
+		ignoredTagsOnUpdate := []string{
+			"#EXTM3U",
+			"#EXT-X-VERSION",
+			"#EXT-X-DISCONTINUITY-SEQUENCE",
+			"#EXT-X-PUBLISHED-TIME",
+			"#EXT-X-TARGETDURATION",
+			"#EXT-X-MEDIA-SEQUENCE",
+			"#EXT-X-PROGRAM-DATE-TIME",
+			"#EXT-X-KEY",
```

Comment:
I'm sure there are a whole bunch of others that you wouldn't want to print out on every manifest refresh.  I was testing against only one manifest for now.